src/github/pullRequestModel.ts

```diff
@@ -31,7 +31,12 @@ export class PullRequestModel implements IPullRequestModel {
 	public localBranchName?: string;
 	public mergeBase?: string;
 	public isDraft?: boolean;
-	public suggestedReviewers: ISuggestedReviewer[];
+	public suggestedReviewers?: ISuggestedReviewer[];
+	public milestone?: IMilestone;
+
+	public get isIssue(): boolean {
```

Comment:
Instead of having an issue be a type of pull request, I think the reverse would be more accurate. Could we have a separate `IssueModel` that `PullRequestModel` would extend?

Reply:
I started down that path initially, but it involves large amounts of refactoring. Would you like me to do it in this PR, or in an immediate follow up PR to keep things easier to review?

Reply:
Splitting it up into a follow up PR sounds good to me 👍

Reply:
That's what I'll do then!